src/screens/CommunityHome/index.tsx

```diff
@@ -119,13 +120,36 @@ export default function CommunityHome({ route }: any) {
     }
   }, [postSetting]);
 
+  const loadCommunity = useCallback(async () => {
+    try {
+      const unsubscribe = CommunityRepository.getCommunity(
+        communityId,
+        // setCommunityData
+        (community) => {
+          setCommunityData(community);
+          setPostSetting(community?.data?.postSetting);
+          if (
+            (community.data as Amity.RawCommunity).needApprovalOnPostCreation
+          ) {
+            setPostSetting('ADMIN_REVIEW_POST_REQUIRED');
+          }
+          setIsJoin(community?.data.isJoined || false); // Set isJoin to communityData?.data.isJoined value
+        }
+      );
+      unsubscribe();
+    } catch (error) {
+      console.error('Failed to load communities:', error);
+    }
+  }, [communityId]);
```

Comment:
we should exclude this change

Reply:
@bonanaaaaaa While running yarn prepare, there were some eslint issues which are preventing a new commit.
Above code is due to useEffect dep. We have to add `loadCommunity` in useEffect. So that I have to memoize with useCallback and it makes a new changes.